src/main/java/chronosacaria/mcdw/client/McdwClient.java

```diff
@@ -197,9 +184,9 @@ public static void registerCrossbowPredicates(McdwCrossbow crossbow) {
         });
     }
 
-    public static void registerShieldPredicates(McdwShield shield){
+    public static void registerShieldPredicates(McdwShield shield) {
         ModelPredicateProviderRegistry.register(shield, new Identifier("blocking"), (itemStack, clientWorld,
-                livingEntity, seed) -> livingEntity != null && livingEntity.isUsingItem() && livingEntity.getActiveItem()
-                == itemStack ? 1.0F : 0.0F );
+                                                                                     livingEntity, seed) -> livingEntity != null && livingEntity.isUsingItem() && livingEntity.getActiveItem()
```

Comment:
This indenting feels a little weird but it looks like there's no good way to do it so meh.

Reply:
That was Idea's autoformatter. Tried to remove it with the last commit but i guess it autoformatted anyways. I will add a fix